### PR TITLE
Fio histo log pctiles

### PR DESCRIPTION
**Pull request by bengland2**
as requested in [issue 624](https://github.com/axboe/fio/issues/624) , this PR has the new method for generating latency percentiles from histogram logs.

### CODE REVIEW

**Comment by axboe** [comment]
Pulled, thanks Ben.

File: tools/hist/fio-histo-log-pctiles.py [hunk 1]
```diff
@@ -307,99 +298,61 @@ def get_pctiles(buckets, wanted, time_ranges):
...
+        help="fio histogram buckets-per-group bits (default=6 means 64 buckets/group)")
+    parser.add_argument("--time-quantum", dest="time_quantum", 
+        default="1", type=int,
+        help="time quantum in seconds (default=1)")
+    parser.add_argument("--output-unit", dest="output_unit", 
+        default="usec", type=str,
+        help="Latency percentile output unit: msec|usec|nsec (default usec)")
+    parser.add_argument("file_list", nargs='+')
+    args = parser.parse_args()
+    print(args)
```
Comment (ndokos) [inline tools/hist/fio-histo-log-pctiles.py]: Is this debugging output? Should it be deleted?

File: tools/hist/fio-histo-log-pctiles.py [hunk 2]
```diff
@@ -307,99 +298,61 @@ def get_pctiles(buckets, wanted, time_ranges):
...
+
+    if not args.bucket_groups:
         # default changes based on fio version
         if fio_version == 2:
```
Comment (ndokos) [inline tools/hist/fio-histo-log-pctiles.py]: Should this be `args.fio_version == 2`?

File: tools/hist/fio-histo-log-pctiles.py (hunk 1 above)
Comment (bengland2) [reply tools/hist/fio-histo-log-pctiles.py]: dang it.  you were right.  Will try to get rid of that next time around.

**Comment by axboe** [comment]
Ben, can you take care of these issues now? Thanks.

**Comment by bengland2** [comment]
was on vacation last 2 business days, should be able to fix Monday.

**Pull request merged**